docker-src/app/Dockerfile

```diff
@@ -7,12 +7,12 @@ RUN apt autoclean && apt update
 RUN apt -y install netcat \
     && apt -y install curl \
     && apt -y install openjdk-8-jdk-headless \
-    && apt -y install postgresql-client-9.5 \
+    && apt -y install postgresql-client-12 \
     && apt -y autoremove \
     && apt -y autoclean
 
 RUN mkdir -p /opt/metasfresh
-RUN curl https://repo.metasfresh.com/repository/mvn-release-releases/de/metas/dist/metasfresh-dist-dist/5.174.2-461+release/metasfresh-dist-dist-5.174.2-461+release-dist.tar.gz \
+RUN curl https://repo.metasfresh.com/repository/mvn-release-releases/de/metas/dist/metasfresh-dist-dist/5.175.2-622+release/metasfresh-dist-dist-5.175.2-622+release-dist.tar.gz \
```

Comment:
our latest *actual* release is still 5.174. this do this update?

Reply:
this 5.174 version isn't available anymore, if you can provide me one that still exists i can change it again